### DISCUSSION TITLE
How to integrate my Maestro tests with BrowserStack to run on specific devices?

### DISCUSSION THREAD

**Discussion by Ah-Lice** [mobile-dev-inc/Maestro#1434] [Q&A]
I have a few test cases written and I'd like to integrate them with BrowserStack. is it possible? Can someone help with the documentation?

**Comment by luflow** [comment]
Same here - we want to use maestro on browserstack. Is there an integration planned?

**Comment by Fishbowler** [comment]
I think they've got something in beta now
